packages/@okta/i18n/src/properties/login.properties

```diff
@@ -513,10 +513,11 @@ enroll.totp.enterCode = Enter code displayed from the application
 
 ## Barcode View
 # {0} is the name of a MFA factor
-enroll.totp.setupApp = Launch {0} application on your mobile device and select Add an account.
+enroll.totp.setupApp = Launch {0} on your mobile device and select “Add an account”. Scan the QR code to continue.
 # {0} is the name of a MFA factor
 enroll.totp.setupGoogleAuthApp = Launch {0}, tap the "+" icon, then select "Scan barcode".
 enroll.totp.cannotScan = Can't scan?
+enroll.totp.aria.cannotScan = Setup without scanning a QR code.
```

Comment:
does this require adding a new key to `i18nTransformer.js`? @nikhilvenkatraman-okta

Reply:
The easiest way to know if we need to add it to i18nTransformer is probably checking the UI by loading it in `ok_PL` language. Given that this is used within the `aria-label` attribute, I am guessing this does not get rendered on the UI.

Reply:
Yes it will not be loaded in UI. when i checked in DOM, i could see the encoded value.

Reply:
I don't think it does, the aria label does not map to any field in the API response.